rules/when-to-use-entity-framework/rule.md

```diff
@@ -54,4 +54,5 @@ Figure: Good example - at least 4 good reasons below
 2. Less performance issues - Most serious .NET performance issues were because of unclosed connections. LINQ means no connection code needed to be done.
```

Comment:
Just remove this point. If you didn't dispose the connection at the end, you used to leak SQLConnections which eventually lead to resource exhaustion. The same is true if you manually create dbcontexts too, the DI handles this for you now, so just don't even mention it. It's not related to this discussion at all.